pkg/controller/constrainttemplate/constrainttemplate_controller_test.go

```diff
@@ -218,20 +218,35 @@ violation[{"msg": "denied!"}] {
 				}
 			}
 			return errors.New("DenyAll not found")
-		}, timeout).Should(gomega.BeNil())
+		})
+		if err != nil {
+			t.Fatal(err)
+		}
 	})
 
 	logger.Info("Running test: Constraint is marked as enforced")
```

Comment:
bring inside t.Run() (or at least have this log lines in a consistent location relative to t.Run())

Reply:
Done, for this and other log messages. Now the log message is only printed if the test is actually run.